lib/PlaidClient.js

```diff
@@ -214,7 +214,13 @@ Client.prototype.getTransactions =
 
 // getAllTransactions(String, Date, Date, Object?, Function)
 Client.prototype.getAllTransactions =
-  function(access_token, start_date, end_date, cb) {
+  function(access_token, start_date, end_date, options, cb) {
```

Comment:
Its a little more work, but really we could support `offset` and `count` here as well, right ? I guess if you want that functionality, you can just use `getTransactions`?

Instead of using `ItemRequestOptions`, can we have something like:

```typescript
  interface GetAlltransactionsRequestOptions extends ItemRequestOptions {}
```

This way, we can add this functionality later without needing to make a breaking change to the typings.

Reply:
I see the purpose of `getAllTransactions` mainly to just get all the transactions between 2 dates (`start_date` and `end_date`), while ignoring any complications around pagination. So yeah, I think if you wanted to specify those parameters, you should just use `getTransactions`. I can't think of any scenario, why you'd want to specify those while using this method.

Will change the interface as suggested.